src/Scorecard.js

```diff
@@ -0,0 +1,36 @@
+function Scorecard() {
```

Comment:
Did you try installing eslint? It helps a lot with them semi colons, whether you needed it or not